webapp/channels/src/components/access_history_modal/access_history_modal.tsx

```diff
@@ -29,15 +29,9 @@ const AccessHistoryModal = ({
     onHide,
     userAudits,
 }: Props) => {
-    const [show, setShow] = useState(true);
-
-    const onCloseClick = useCallback(() => {
-        setShow(false);
-    }, []);
-
     useEffect(() => {
         getUserAudits(currentUserId, 0, 200);
-    }, []);
+    }, [currentUserId, getUserAudits]);
```

Comment:
Similar to my other comment on an effect. This is a "potential change in behavior".